src/reanimated2/platform-specific/checkVersion.ts

```diff
@@ -1,7 +1,33 @@
-import { checkCppVersion } from './checkCppVersion';
-import { checkPluginVersion } from './checkPluginVersion';
+import { jsVersion } from './jsVersion';
 
-export function checkVersion(): void {
+export function checkVersion() {
   checkCppVersion();
-  checkPluginVersion();
+}
```

Comment:
Can we remove `checkVersion` function and export `checkCppVersion` function directly?

Reply:
Sure ting!